docs/traits.md

```diff
@@ -91,35 +91,31 @@ struct Access<NearestToOrigin, PredicatesTag>
 
 int main(int argc, char *argv[])
 {
-  Kokkos::initialize(argc, argv);
-  {
-
-    constexpr std::size_t N = 1000;
-    std::array<float, N> a;
+  Kokkos::Kokkos::ScopeGuard guard(argc, argv);
```

Comment:
```suggestion
  Kokkos::ScopeGuard guard(argc, argv);
```

Reply:
Alternatively I can try to sneak `namespace Kokkos { using Kokkos; }` in `Kokkos_Core.hpp`